logging_manager/src/main.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import time
+import glob
+import os
+
+# Call StorageManager
+exec(open("StorageManager.py").read())
+
+# Check if acknowledgment is received from StorageManager
+if "done" in globals():
```

Comment:
if you have a main function in storage manager, you can call that from here with an import, and track the return value directly without needing to check for something in globals()